Subject: ui: details: fix Applications width.
From: nose-gnome 71675376+nose-gnome@users.noreply.github.com Thu Mar 23 15:55:42 2023 +0000
Date: Sat Mar 2 14:19:28 2024 -0500:
Git: 04cc4895298c4eb86394a9840f2a6a8161b5a54d

In OS info the applications tree view was too narrow to read the contents, so I enabled expanding horizontally for scrolledwindow6

diff --git a/ui/details.ui b/ui/details.ui
index 180abe0c..29ed119b 100644
--- a/ui/details.ui
+++ b/ui/details.ui
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.38.2 -->
+<!-- Generated with glade 3.40.0 -->
 <interface>
   <requires lib="gtk+" version="3.22"/>
   <object class="GtkAdjustment" id="adjustment1">
@@ -791,7 +791,9 @@
                                 <property name="height-request">150</property>
                                 <property name="visible">True</property>
                                 <property name="can-focus">True</property>
+                                <property name="hexpand">True</property>
                                 <property name="shadow-type">etched-in</property>
+                                <property name="propagate-natural-width">True</property>
                                 <child>
                                   <object class="GtkTreeView" id="inspection-apps">
                                     <property name="visible">True</property>
